cmake/modules/HexagonSDK.cmake

```diff
@@ -97,11 +132,26 @@ function(_get_hexagon_sdk_property_impl
     endif()
 
   elseif(_property STREQUAL "QAIC_EXE")
-    _check_path_exists(
-      "${_hexagon_sdk_root}/ipc/fastrpc/qaic/Ubuntu18/qaic"
-      _qaic_path
-    )
-    set_parent(${_output_variable} "${_qaic_path}")
+    set(_override $ENV{QAIC_PATH_OVERRIDE})
+    if(_override)
+      set_parent(_qaic_path "${_override}")
+    else()
+      _get_ubuntu_version(_uversion)
+      _check_path_exists(
+        "${_hexagon_sdk_root}/ipc/fastrpc/qaic/${_uversion}/qaic"
+        _qaic_path
+      )
+    endif()
+    _check_path_exists("${_qaic_path}" _qaic_path_found)
+    if(NOT _qaic_path_found)
+      message(
+        SEND_ERROR
```

Comment:
Is there a particular reason to use `SEND_ERROR` instead of `FATAL_ERROR`?

Reply:
Changed it to a WARNING---the message is there to inform the user about the override flag.